ApiReview/API.Polly.Core/NoDocs/Polly.Fallback/OnFallbackArguments.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Polly.Fallback;
+
+public record OnFallbackArguments
```

Comment:
Consider using standard `class` to not expose additional APIs, we don't use these in dictionaries nor don't compare them.

Reply:
True, this is already done.